autoencoder.py

```diff
@@ -0,0 +1,72 @@
+"""Implementation of feed forward autoencoder in TensorFlow 2.0"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__version__ = '1.0.0'
+__author__ = 'Richard Ricardo'
+
+import tensorflow as tf
+from tensorflow.keras.layers import Dense
+from keras.models import Model
```

Comment:
Why are we importing the `keras.models.Model` when we are using `tf.keras.Model`?

Reply:
My bad, I just realize I don't need it. Thank you for this.